app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonDatabaseAdapter.java

```diff
@@ -174,6 +184,21 @@ public void updateContact(final Contact oldContact, final Contact newContact) th
      * {@inheritDoc}
      */
     @Override
+    public Contact getContactWithName(final String contactName) throws IOException {
+        final List<Contact> contacts = getAllContacts();
+        for (final Contact contact: contacts) {
+            if(contact.getName().equals(contactName)) {
+                return contact;
+            }
+        }
+
+        return null;
```

Comment:
Maybe it's hadny to put this up as a sticky note or something in this pr/our project as other classes that will depend on this need to be aware of this return strategy.